SPECS-SIGNED/mft_kernel-signed/mft_kernel-signed.spec

```
@@ -31,6 +29,11 @@ ExclusiveArch:  x86_64
 Requires:       kernel = %{target_kernel_version_full}
 Requires:       kmod
 
+# Best practice for specs mandate package name and rpm name must
+# match. Default mft_kernel spec set rpm name as kernel-mft. To
+# comply, we set Name as mft_kernel but add a Provides for kernel-mft.
+Provides:       kernel-mft
```

Please always add version information for `Provides`, so that each new version provides its own. Otherwise, if you try to install a specific version of `kernel-mft`, the package manager may not be able to find it.
```suggestion
Provides:       kernel-mft = %{version}-%{release}
```

---

Yes, will add that.